pkg/kubectl/cmd/util/factory_client_access.go

```diff
@@ -454,8 +454,8 @@ const (
 	SecretForDockerRegistryV1GeneratorName      = "secret-for-docker-registry/v1"
 	SecretForTLSV1GeneratorName                 = "secret-for-tls/v1"
 	ConfigMapV1GeneratorName                    = "configmap/v1"
-	ClusterRoleBindingV1GeneratorName           = "clusterrolebinding.rbac.authorization.k8s.io/v1alpha1"
```

Comment:
stability of output meant: this use to give me a replication controller but now it gives me a replica set.

at the time, it did not mean, this use to give me a particular version of that kind.

it seems like sig-cli should decide how to proceed here given that --output-version seems to not work anymore either...